azure-pipelines.yml

```diff
@@ -296,4 +297,35 @@ jobs:
       - task: CmdLine@2
         displayName: 'Build Community Toolkit Sample'
         inputs:
-          script: 'dotnet build $(PathToCommunityToolkitSampleCsproj) -c Release'
\ No newline at end of file
+          script: 'dotnet build $(PathToCommunityToolkitSampleCsproj) -c Release'
+
+  - job: verify_dotnet_format
+    displayName: Verify Code Formatting
+    pool:
+      vmImage: macos-12
+    steps:
+      - task: CmdLine@2
+        displayName: 'Set Xcode Version'
+        inputs:
+          script: echo '##vso[task.setvariable variable=MD_APPLE_SDK_ROOT;]'/Applications/Xcode_$(XcodeVersion).app;sudo xcode-select --switch /Applications/Xcode_$(XcodeVersion).app/Contents/Developer
+      
+      - task: UseDotNet@2
+        displayName: Install Latest .NET SDK
+        inputs:
+          packageType: 'sdk'
+          version: '$(LATEST_NET6_VERSION)'
+
+      - task: CmdLine@2
+        displayName: 'Install Latest .NET MAUI Workload'
```

Comment:
You don’t need MAUI to run dotnet format

Reply:
I thought that that `dotnet format` does use `dotnet build` under the hood?

If so, then wouldn't we need the .NET MAUI Workload?

Reply:
I don't see any reason to run build on formatting. Just need to check.

If it doesn't run build then we can run this job only on one os

Reply:
seems like you are right. Approved